UnitTest.Rollbar/RollbarLoggerBlockingWrapperFixture.cs

```diff
@@ -0,0 +1,87 @@
+﻿#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+
```

Comment:
Any tests that use actual time and timeouts are fragile and usually force tests to take longer than they need to. Testing timeout behaviour should be possible without having to actually wait for things to timeout or not. These tests will fail at some point when something takes longer to run that it usually does. We need a different way of injecting time to be able to test this better.

Reply:
agree. but here we are waiting for the expected timeout of a blocking call while also making sure that that it is not broken and the timeout exception is thrown. btw, we do have a few other unit test methods protected by a timeout attribute just in case to avoid "hanging" of the test in case the unit under test malfunctions. i normally set these guarding timeout as multiples of "tested timeout" somewhat unreasonably large for the tested scenario. so, if it does fail due to the delayed over time execution logic, we probably do want to look into it anyway.